apps/demo/src/Views/Ui/index.tsx

```diff
@@ -52,6 +57,11 @@ const componentList = [
     title: "buttons.export",
     component: <ExportButton />,
   },
+  {
+    key: 6,
+    title: "files.table",
+    component: <FilesTable files={[]} />,
```

Comment:
Add some dummy data here instead of empty [] so that we get the glimpse of what table will look like with data.